code/modules/rituals/ritual_types/divine/ritualcircles.dm

```diff
@@ -0,0 +1,290 @@
+		if("Moonlight Dance")
+			if(do_after(user, 50))
+				user.say("I beseech the he-form of the Twinned God!!")
+				if(do_after(user, 50))
+					user.say("To bring Wisdom to a world of naught!!")
+					if(do_after(user, 50))
+						user.say("Place your gaze upon me, oh wise one!!")
+						to_chat(user,span_cultsmall("The waning half of the Twin-God carries but one eye. With some effort, it can be drawn upon supplicants."))
+						playsound(loc, 'sound/magic/holyshield.ogg', 80, FALSE, -1)
+						moonlightdance(src)
+						user.apply_status_effect(/datum/status_effect/debuff/ritesexpended)
+
+/obj/structure/ritualcircle/noc/proc/moonlightdance(src)
+	var/ritualtargets = view(7, loc)
+	for(var/mob/living/carbon/human/target in ritualtargets)
+		target.apply_status_effect(/datum/status_effect/buff/moonlightdance)
+
+/obj/structure/ritualcircle/xylix
+	name = "Rune of Trickery"
+	desc = "A Holy Rune of Xylix"
+
+/obj/structure/ritualcircle/ravox
+	name = "Rune of the Warrior"
+	desc = "A Holy Rune of Ravox"
+
+/obj/structure/ritualcircle/pestra
+	name = "Rune of Plague"
+	desc = "A Holy Rune of Pestra"
+	icon_state = "pestra_chalky"
+	var/plaguerites = list("Flylord's Triage")
+
+
+/obj/structure/ritualcircle/pestra/attack_hand(mob/living/user)
+	if((user.patron?.type) != /datum/patron/divine/pestra)
+		to_chat(user,span_smallred("I don't know the proper rites for this..."))
+		return
+	if(!HAS_TRAIT(user, TRAIT_RITUALIST))
+		to_chat(user,span_smallred("I don't know the proper rites for this..."))
+		return
+	if(user.has_status_effect(/datum/status_effect/debuff/ritesexpended))
+		to_chat(user,span_smallred("I have performed enough rituals for the day... I must rest before communing more."))
+		return
+	var/riteselection = input(user, "Rituals of Plague", src) as null|anything in plaguerites
+	switch(riteselection) // put ur rite selection here
+		if("Flylord's Triage")
+			if(do_after(user, 50))
+				user.say("Buboes, phlegm, blood and guts!!")
+				if(do_after(user, 50))
+					user.say("Boils, bogeys, rots and pus!!")
+					if(do_after(user, 50))
+						user.say("Blisters, fevers, weeping sores!!")
+						to_chat(user,span_danger("You feel something crawling up your throat, humming and scratching..."))
+						if(do_after(user, 30))
+							icon_state = "pestra_active"
+							user.say("From your wounds, the fester pours!!")
+							to_chat(user,span_cultsmall("My devotion to the Plague Queen allowing, her servants crawl up from my throat. Come now, father fly..."))
+							loc.visible_message(span_warning("[user] opens their mouth, disgorging a great swarm of flies!"))
+							playsound(loc, 'sound/misc/fliesloop.ogg', 100, FALSE, -1)
+							flylordstriage(src)
+							user.apply_status_effect(/datum/status_effect/debuff/ritesexpended)
+							spawn(120)
+								icon_state = "pestra_chalky"
+
+/obj/structure/ritualcircle/pestra/proc/flylordstriage(src)
+	var/ritualtargets = view(0, loc)
+	for(var/mob/living/carbon/human/target in ritualtargets)
```

Comment:
```suggestion
	for(var/mob/living/carbon/human/target in view(0, src))
```
don't use a var for it, but also you could just do `in loc` if you're just doing view(0).